From 030717c0d442b21328874d6af42f9c6f89d2d120 Mon Sep 17 00:00:00 2001
From: Jo Zzsi <jozzsicsataban@gmail.com>
Date: Thu, 5 Sep 2024 09:09:36 -0400
Subject: [PATCH 8/8] fix(fips-crypto-policies): make it depend on fips dracut
 module

(cherry picked from commit a2096dafdbfc88eed91ce34b1f4d27e7eb7ca839)

Conflicts:
      modules.d/01fips-crypto-policies/module-setup.sh
      Due to upstream e6117b92fa0108dbaf9ea3ac0ec8f5a02487c812, which
      was not cherry-picked. Resolved the conflict by keeping the
      functions (i.e., undoing the cleanup of the upstream commit).
---
 modules.d/01fips-crypto-policies/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/01fips-crypto-policies/module-setup.sh b/modules.d/01fips-crypto-policies/module-setup.sh
index ee00452e..140eae00 100755
--- a/modules.d/01fips-crypto-policies/module-setup.sh
+++ b/modules.d/01fips-crypto-policies/module-setup.sh
@@ -11,6 +11,7 @@ check() {
 
 # called by dracut
 depends() {
+    echo fips
     return 0
 }
 
-- 
2.42.0

